inginious/common/course_user_setting.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
This whole thing in a frontend layer that the backend should not be aware of. Please move those files from ``/common`` to ``/frontend/user_settings`` for instance, as you called it for the templates.